.github/workflows/scripts/spdx-tools-java-wrapper.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Maximilian Huber <maximilian.huber@tngtech.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+set -euo pipefail
+
+version="1.1.3"
+url="https://github.com/spdx/tools-java/releases/download/v${version}/tools-java-${version}.zip"
+jar="$HOME/spdx-tools-java/tools-java-${version}-jar-with-dependencies.jar"
+
+bootstrap() {
+    if [[ ! -f "$jar" ]]; then
+        mkdir -p "$(dirname "$jar")"
+        local tmpdir="$(mktemp -d)"
+        (cd "$tmpdir"
+        wget -q "$url"
+        unzip "tools-java-${version}.zip"
+        )
+        mv "$tmpdir/$(basename "$jar")" "$jar"
+
+        java -jar "$jar"
+    fi
+}
+
+verify() {
+    local tmpfile="$(mktemp)"
+    find analysed-packages/ -iname '*.spdx' -print0 |
+        while IFS= read -r -d '' spdx; do
+            java -jar "$jar" Verify "$spdx" || {
+                echo "$spdx" >> "$tmpfile"
+            }
+        done
```

Comment:
Can this be simplified via `xargs`?

Reply:
maybe, but might need another hack. Since the command that is executed within the while loop is more then just one cli call.